DFC.ServiceTaxonomy.ApiFunction/Function/Execute.cs

```diff
@@ -0,0 +1,166 @@
+using DFC.ServiceTaxonomy.ApiFunction.Helpers;
+using DFC.ServiceTaxonomy.ApiFunction.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web.Http;
+using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+namespace DFC.ServiceTaxonomy.ApiFunction.Function
+{
+    public class Execute
+    {
+        private readonly IOptions<ServiceTaxonomyApiSettings> _serviceTaxonomyApiSettings;
```

Comment:
`IOptions` is fine, but Microsoft seem to [suggest using `IOptionsMonitor` in new code](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-3.0).